tests/PhpPact/Plugins/Protobuf/Service/GrpcMockServerTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace PhpPactTest\Plugins\Protobuf\Service;
+
+use PhpPact\Plugins\Protobuf\Service\GrpcMockServer;
+use PhpPactTest\Consumer\Service\MockServerTest;
+
+class GrpcMockServerTest extends MockServerTest
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->mockServer = new GrpcMockServer($this->client, $this->pactDriver, $this->config);
+    }
+
+    protected function getTransport(bool $secure): string
+    {
+        return 'grpc';
+    }
```

Comment:
I'm confused by this as well. I Thought gRPC had to use https and HTTP2

Reply:
I don't understand much about the pact's plugin system. Here is what I "know":

`Transports` is one of a few **new terms** defined by Pact core when implementing [plugins](https://docs.pact.io/plugins/quick_start#introduction-to-plugins).

There are 3 transports so far that I know:
* core/transport/http
* core/transport/https
* plugin/protobuf/transport/grpc

All I know is: when I need to use grpc mock server, I need to use `grpc` transport provided by protobuf plugin, not `http` or `https` provided by the core.